index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+			<title>Calendar</title>
+	<link rel="stylesheet" href = "style.css">
+	<script src="script.js" defer></script>
+	</head>
+
+	<body id = "body">
+		<H3 class = "heading"> Calendar</H3>
+
+		<div class=" date-select-division">
+			<!--Dropdown -->
+			<label >Month:</label>
+
+				<select name="Months" id="Months">
```

Comment:
index.html should have only body tag. Rest all should be added dynamically using JS.